localStorage.js

```diff
@@ -0,0 +1,24 @@
+const makeInput = document.getElementById('make');
+const emailInput = document.getElementById('email');
+
+const storageInfo = JSON.parse(localStorage.getItem('storageInfo'));
+if (storageInfo) {
+  makeInput.value = storageInfo.make;
+  emailInput.value = storageInfo.email;
+}
+document
+  .getElementById('register-form')
+  .addEventListener('submit', () => {
+    const make = makeInput.value.trim();
+    const email = emailInput.value.trim();
+
+    if (!make || !email) {
+      return;
+    }
+
+    const storageInf = {
+      email,
+      make,
+    };
+    localStorage.setItem('storageInfo', JSON.stringify(storageInf));
+  });
```

Comment:
Great job storing the values after submitting the form! Although, as requested in the project requirements: 
![image](https://user-images.githubusercontent.com/42799579/202735197-59bfa015-d912-45ff-b6bb-2c9234d9d97a.png) 

Kindly notice that the user data should be stored after any changes to the form, not after submitting it 😉. 

Also, please notice that the message field isn't being stored, only the email + username 😄:
![image](https://user-images.githubusercontent.com/42799579/202742426-38e891bf-80b0-4691-9fd5-9947ea8e216d.png)

Kindly adjust it so all fields can be stored 😉.